go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/prometheus/prometheus
 
-go 1.20
+go 1.21.7
```

Comment:
This should probably stay as `1.21`

Reply:
- fixed. https://github.com/prometheus/prometheus/pull/13604/commits/e171f6c8aa50da4dea2b6a27a5c126a304613fe8